README.md

```diff
@@ -190,6 +191,46 @@ class TritonPythonModel:
     that is created must have "TritonPythonModel" as the class name.
     """
 
+    @staticmethod
+    def auto_complete_config(model_config):
+        """`auto_complete_config` is called only once when the server is started
+        with `--strict-model-config=false`. Implementing this function is optional.
+        A no implementation of `auto_complete_config` will do nothing. This function
+        can be used to set `max_batch_size`, `input` and `output` properties of the
+        model using `pb_utils.set_max_batch_size`, `pb_utils.add_input`, and
+        `pb_utils.add_output`. These properties will allow Triton to load the model
+        with minimal model configuration in absence of a configuration file. This
+        function returns the `pb_utils.ModelConfig` object with these properties.
+
+        Note: All the objects in this function will go out of scope after exiting.
+        Should not store any objects in this function.
+
+        Parameters
+        ----------
+        model_config : string
```

Comment:
How would they refer to the config then? As we don't have any accessor functions on pb_utils.ModelConfig .

I think this was added so that users can refer to the current config and determine the inputs/outputs and max_batch_size.

Reply:
How about having a `as_dict` that provides a read-only access to the model configuration? cc @krishung5

Reply:
I think using `as_dict` to provide the model configuration sounds good to me.